Explorer/Assets/DCL/SDKComponents/CameraControl/CameraModeArea/Systems/CameraModeAreaHandlerSystem.cs

```diff
@@ -14,23 +12,27 @@
 using ECS.LifeCycle;
 using ECS.LifeCycle.Components;
 using ECS.Unity.Transforms.Components;
+using System.Collections.Generic;
 
 namespace DCL.SDKComponents.CameraModeArea.Systems
 {
     [UpdateInGroup(typeof(SyncedInitializationFixedUpdateThrottledGroup))]
-    [UpdateBefore(typeof(CharacterTriggerAreaCleanUpRegisteredCollisionsSystem))]
     [LogCategory(ReportCategory.CAMERA_MODE_AREA)]
     public partial class CameraModeAreaHandlerSystem : BaseUnityLoopSystem, IFinalizeWorldSystem
     {
-        private static CameraMode cameraModeBeforeLastAreaEnter; // There's only 1 camera at a time
+        // There's only 1 camera at a time
+        private static CameraMode cameraModeBeforeLastAreaEnter;
+        private static readonly HashSet<Entity> activeAreas = new HashSet<Entity>();
```

Comment:
Uhmmm that hashset was made `static` so that only 1 instance of it exists, that's because, even though there can be many `CameraModeAreaHandlerSystem` instances (in different scene worlds):
- There can only be 1 **active** camera mode area **effect** at a time
- The camera mode areas are only activated by the main player, and the main player can't be in different scenes at the same time

That's why for me it makes sense to have only 1 static collection, if we remove the `static` (to pool it) it will exists unnecessarily for CameraModeArea in non-current scenes...

Anyway I expanded the comment in there so that it's more clear, WDYT? done at https://github.com/decentraland/unity-explorer/pull/1744/commits/5df8bb68274eb58ee1ec408ec5da70c7e814c19a

Reply:
Well, I didn't notice that it's static, though I think it's a bit error-prone